natural-sql-7b/config.yaml

```diff
@@ -0,0 +1,25 @@
+environment_variables: {}
+external_package_dirs: []
+model_framework: transformers
+model_type: AutoModelForCausalLM
+description: NaturalSQL-7B is a state-of-the-art model for generating SQL queries from natural language questions, developed by ChatDB.
+model_metadata:
+  avatar_url: https://cdn-uploads.huggingface.co/production/uploads/648a374f00f7a3374ee64b99/5ynfoKPzI3_-WasQQt7qR.png
+  cover_image_url: https://global-uploads.webflow.com/6348b2d49808811e3f7a0fff/640690727b722a05771960ec_graphic-data-p-800.png
+  tags:
+  - text-to-SQL
+  - natural language processing
+  - SQL generation
```

Comment:
These tags are made up. For example our [nsql](https://github.com/basetenlabs/truss-examples/blob/main/nsql/config.yaml) truss has only one tag `code-generation`